packages/11ty/content/_assets/javascript/application/canvas-panel.js

```diff
@@ -180,29 +187,31 @@ const setUpUIEventHandlers = () => {
 }
 
 /**
- * Update canvas panel properties
+ * Update canvas panel or image-service properties
  * 
- * @param  {String} canvas id
+ * @param  {String} id Canvas ID or path to image-service info.json
  * @param  {Object} data
  * @property {String} region comma-separated, @example "x,y,width,height"
  * @property {Array<Object>} annotations
  */
-const update = (canvasId, data) => {
-  const canvasPanels = document.querySelectorAll(`[canvas-id="${canvasId}"]`)
-  if (!canvasPanels.length) {
-    console.error(`Failed to call update on canvas with id ${canvasId}. Canvas does not exist.`)
+const update = (id, data) => {
+  const serviceComponents = document.querySelectorAll(`canvas-panel[canvas-id="${id}"], image-service[src="${id}"]`)
+  if (!serviceComponents.length) {
+    console.error(`Failed to call update on canvas panel or image-service component with id ${id}. Element does not exist.`)
   }
   const { annotations, region } = data
-  canvasPanels.forEach((canvasPanel) => {
+  serviceComponents.forEach((element) => {
```

Comment:
Elsewhere we are naming these `webComponents`, we might do so here as well, or perhaps simply `components`?